### DISCUSSION TITLE
m3u8

### DISCUSSION THREAD

**Discussion by Brirringo** [anasty17/mirror-leech-telegram-bot#1003] [Q&A]
Ok, to provide context for Coffe's conversation:

The m3u8 link I'm trying to download is from Dailymotion:
https://proxy-048.ix7.dailymotion.com/sec(Ag2kWK5ZJFfHITwn7VOjZLrONS3ZZHlwGmgb3r2vbWp6zunS1LFLg_1Fiioc7xk62PbN3AwrD1g25HlQhoajYKtEaKRt2Gb8lzMYrt8ju6A)/video/555/401/506104555_mp4_h264_aac_hd.m3u8 (The link lasts a few hours, so it may not be available after that)

I send the link to another bot and it starts downloading
![Screenshot_1](https://user-images.githubusercontent.com/38145414/187108720-936b7a0c-06ed-4e38-a212-1353007250ce.png)

But when I send it to my bot (based on your repository), it just gives me HTTP Error 403
![Screenshot_2](https://user-images.githubusercontent.com/38145414/187109000-802501dc-1197-4495-a064-ce1a9e253470.png)

You may wonder why I don't copy the Dailymotion link and send it with /ytdl to the bot, but the video can only be seen from the page where the video is embedded.

**Comment by anasty17** [comment]
That's conversation was useless. Yeah seems issue from yt-dlp version. Cz the other bot use yt-dlp to download it! Let me make some tests and then if something wrong in my repo i will fix it. If not so i will report to yt-dlp.
Implementation of m3u8 is very easy, but why adding m3u8 downloader and we already have yt-dlp!

**Reply by Brirringo** [reply]
The [page](https://www.canalrcn.com/la-rosa-de-guadalupe/capitulos/la-rosa-de-guadalupe-l-capitulo-12-l-una-mujer-marchita-3027) that embeds the Dailymotion video is only available in my country and only the video can be viewed within that page.

So, Ithink the blocking is part of Dailymotion:
https://github.com/ytdl-org/youtube-dl/issues/20961
https://github.com/yt-dlp/yt-dlp/issues/3032

**Reply by anasty17** [reply]
As i said there no issue in yt-dlp.  403 means site blocked current location. I have tried it locally and it's working! So it's about host location.

**Reply by anasty17** [reply]
![Screenshot_20220829-062714_Termux](https://user-images.githubusercontent.com/77075674/187117279-ab88cdb5-a008-44e4-85e7-a9f4ae4b7bee.jpg)

With same repository options.
